code/modules/surgery/organs/internal/cyberimp/augments_eyes.dm

```diff
@@ -59,6 +59,22 @@
 	HUD_type = DATA_HUD_MEDICAL_ADVANCED
 	HUD_trait = TRAIT_MEDICAL_HUD
 
+/obj/item/organ/internal/cyberimp/eyes/hud/medical/Insert(mob/living/carbon/eye_owner, special = FALSE, movement_flags)
+	. = ..()
+	ADD_TRAIT(eye_owner, TRAIT_SURGICAL_TRAINING, ORGAN_TRAIT)
+
+/obj/item/organ/internal/cyberimp/eyes/hud/medical/Remove(mob/living/carbon/eye_owner, special, movement_flags)
+	if (toggled_on)
+		REMOVE_TRAIT(eye_owner, TRAIT_SURGICAL_TRAINING, ORGAN_TRAIT)
+	. = ..()
```

Comment:
I believe these should be `on_mob_insert` and `on_mob_remove`

Reply:
I was going off of the parent implementation that disabled the HUD in a similar fashion. Confirm that I should change them to /on_mob_insert and /on_mob_remove instead?

Reply:
I think so but strictly speaking it's not necessary for this PR

Reply:
since the proc definition of `/on_mob_insert` and ` /on_mob_remove` call for any `.organ_traits` to be added, I deleted my Insert and Remove redefs and added the surgical training to `.organ_traits`. With the redef I made to the `toggle_hud`, it will still get removed or readded if they toggle it.